starter_code/src/movies.js

```diff
@@ -1,20 +1,124 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
 
+function turnHoursToMinutes(array){
+    var newArray = array.map((a)=>{
+        var b = {};
+        for(key in a){
+            b[key] = a[key];
+            b.duration = toMinutes(a.duration);
+            return b;
+        }
+    });
+
+    return newArray;
+}
+
+function toMinutes(str){
+    if (!str.includes('h')) {
+        var point = str.indexOf('min');
+        var min = Number(str.substring(0,point));
+        var hour = 0;
```

Comment:
Be careful with variables' names. Your problem here is calling hour and hours the same variable.